chromium_src/components/policy/tools/template_writers/writer_configuration.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The Brave Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
I believe the license header is wrong (in all new files). It should mention MPL:
```
# Copyright (c) 2022 The Brave Authors. All rights reserved.
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this file,
# You can obtain one at https://mozilla.org/MPL/2.0/.
```

Reply:
Good catch. I copied this from somewhere without reading it properly. Thanks. Resolved in 4b50184b0fde2f26618b0de445dcba5943a4cad2.